src/pages/workspace/WorkspaceProfileCurrencyPage.tsx

```diff
@@ -20,7 +20,7 @@ function WorkspaceProfileCurrencyPage({policy}: WorkspaceProfileCurrencyPageProp
 
     const onSelectCurrency = (item: CurrencyListItem) => {
         Policy.updateGeneralSettings(policy?.id ?? '', policy?.name ?? '', item.currencyCode);
-        Navigation.goBack();
+        Navigation.setNavigationActionToMicrotaskQueue(Navigation.goBack);
```

Comment:
NAB but same here. Let us add a test case for this to avoid possible regression.